frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/components/DesktopInbox.js

```
@@ -41,35 +45,83 @@ const DesktopInbox = ({ tableConfig, filterComponent, ...props }) => {
-      />
+      <div style={tableContainer}>
+        <table style={tableStyle}>
+          <thead>
+            <tr>
+              <th style={thStyle}>Applicant Name</th>
+              <th style={thStyle}>Registration No./Temporary ID</th>
+              <th style={thStyle}>Submission Date</th>
+              <th style={thStyle}>Type</th>
+              <th style={thStyle}>Status</th>
+              <th style={thStyle}>Property ID</th>
+              <th style={thStyle}></th>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)